packages/db/test/config.test.js

```diff
@@ -318,3 +319,65 @@ test('should set migrations table to versions if there are no postgresql schema'
 
   equal(savedConfig.migrations.table, 'versions')
 })
+
+test('config reloads from a written file', async ({ teardown, equal, pass, same }) => {
+  const config = join(os.tmpdir(), `some-config-${process.pid}-2.json`)
+  const file = join(os.tmpdir(), `some-plugin-${process.pid}-2.js`)
+
+  await writeFile(config, JSON.stringify({
+    server: {
+      hostname: '127.0.0.1',
+      port: 0
+    },
+    plugin: {
+      path: file,
+      options: {
+        message: 'hello'
+      }
+    },
+    core: {
+      ...connInfo
+    },
+    metrics: false
+  }))
+
+  await writeFile(file, `
+    module.exports = async function (app, options) {
+      app.get('/', () => options.message)
+    }`)
+
+  const server = await buildServer(config)
+  teardown(server.stop)
```

Comment:
I thought this needed to be bound.

Reply:
`stop` come from restartable, not Fastify and it does not need to be bound.